tests/parametric/test_128_bit_traceids.py

```diff
@@ -119,7 +119,6 @@ def test_datadog_128_bit_propagation_tid_chars(self, test_agent, test_library):
         assert dd_p_tid is None
         assert "x-datadog-tags" not in headers or "_dd.p.tid=" not in headers["x-datadog-tags"]
 
-    @missing_feature(context.library == "dotnet", reason="Optional feature not implemented")
```

Comment:
This one is failing : https://github.com/DataDog/system-tests/actions/runs/9619446629/job/26535678680?pr=2587#step:6:188

Reply:
Weird. I though it was XPASS in the dashboard and I ran it locally, too. Maybe I got the wrong one. Thanks!

Reply:
I reverted this change.